ajax/libs/image-map-resizer/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "image-map-resizer",
+  "version": "1.0.3",
+  "filename": "imageMapResizer.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/davidjbradshaw/image-map-resizer.git"
+  },
+  "description": "Make the HTML image map responsive, so the map scales with the image",
+  "github": "https://github.com/davidjbradshaw/image-map-resizer",
+  "keywords": [
+    "imagemap",
+    "map",
+    "image",
+    "Resizing",
+    "Resizer",
+    "resize",
+    "RWD",
+    "responsive",
+    "jquery-plugin"
+  ],
+  "license": "MIT",
+  "npmName": "image-map-resizer",
+  "npmFileMap": [
+    {
+      "basePath": "js",
```

Comment:
@extend1994 we won't take `js` as base path as we never know if there may be a `css` folder in future, than we must change the structure and break the mainfile(`filename` in package.json) entry point, so let's use `""` empty basePash, and use `js/**/imageMapResizer*` or  `+(js|css)/**/imageMapResizer*` for the files rule.